.gitattributes

```diff
@@ -1,2 +1,3 @@
 frontend/* linguist-vendored
 VERSION export-subst
+* text=auto eol=lf
```

Comment:
This should not go into this PR I guess?

Reply:
yeah, without it I was unable to clone the repo in widows. If you want i could create a new PR, The change is minimal so it should be doable using the github build-in editor